low-level/sspl-ll.spec

```diff
@@ -107,17 +111,14 @@ mkdir -p /var/%{product_family}/sspl/bundle /var/log/%{product_family}/sspl /etc
 SSPL_DIR=/opt/seagate/%{product_family}/sspl
 
 [ -d "${SSPL_DIR}" ] && {
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d /usr/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d $SSPL_DIR/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view /usr/bin/resource_health_view
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate /usr/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate $SSPL_DIR/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/manifest_support_bundle /usr/bin/manifest_support_bundle
+    ln -sf $SSPL_DIR/low-level/sspl_ll_d /usr/bin/sspl_ll_d
```

Comment:
Dont do this. No sspl binaries, scripts should be directly accessible by user on deployment. All needs to be used with absolute path only.
Remove similar existing linking from line 119,120,121

Reply:
Done